src/components/pie-chart.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from "react"
+
+import { ResponsivePie } from "@nivo/pie"
+
+import "./pie-chart.scss"
+
+const COLORS = [
```

Comment:
So does Nivo not handle the colors? Or it doesn't animate them when you hover? I haven't done anything custom with color. I've only used the default Nivo themes, so not sure. Are you basically setting a different transparency when the node is active?

Reply:
Yea, I wasn't able to get it to animate on hover so I'm setting the colors for each node on every re-render in order to adjust the transparency. I think Pie handles colors in a way similar to SankeyChart - you can set a scheme in the colors property, and I copied one of those for this.

I have a sense that I'm overlooking something that's blocking both the color animations and the tooltip...